tools/class-parse/class-parse.csproj

```diff
@@ -12,11 +12,12 @@
     <ProductVersion>8.0.30703</ProductVersion>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
+  <Import Project="..\..\Configuration.props" />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\bin\Debug</OutputPath>
+    <OutputPath>$(UtilityOutputPath)</OutputPath>
```

Comment:
All those projects are *already* using the same value: `..\..\bin\$(Configuration)`. I don't see how a value being shared by multiple projects at all requires that `$(UtilityOutputPath)` be absolute.

I don't believe that this property *must* be an absolute path.